docs/companion.md

```diff
@@ -3,3 +3,716 @@ sidebar_position: 4
+
+#### `customProviders`
+
+This option enables you to add custom providers along with the already supported providers.
+See [adding custom providers](#how-to-add-custom-providers) for more information.
+
+#### `logClientVersion`
+
+A boolean flag to tell Companion whether to log its version upon startup.
+
+#### `metrics` `COMPANION_HIDE_METRICS`
+ 
+A boolean flag to tell Companion whether to provide an endpoint `/metrics` with Prometheus metrics (by default metrics are enabled.)
+
+#### `streamingUpload` `COMPANION_STREAMING_UPLOAD`
+
+A boolean flag to tell Companion whether to enable streaming uploads.
+If enabled, it will lead to _faster uploads_ because companion will start uploading at the same time as downloading using `stream.pipe`.
+If `false`, files will be fully downloaded first, then uploaded. Defaults to `false`,
+but we recommended enabling it, especially if you’re expecting to upload large files.
+In future versions the default might change to `true`.
+
+#### `maxFileSize` `COMPANION_MAX_FILE_SIZE`
+
+If this value is set, companion will limit the maximum file size to process.
+If unset, it will process files without any size limit (this is the default).
+
+#### `periodicPingUrls` `COMPANION_PERIODIC_PING_URLS`
+
+If this value is set, companion will periodically send POST requests to the specified URLs.
+Useful for keeping track of companion instances as a keep-alive.
+
+#### `periodicPingInterval` `COMPANION_PERIODIC_PING_INTERVAL`
+
+Interval for periodic ping requests (in ms).
+
+#### `periodicPingStaticPayload` `COMPANION_PERIODIC_PING_STATIC_JSON_PAYLOAD`
+
+A `JSON.stringify`-able JavaScript Object that will be sent as part of the JSON body in the period ping requests.
+
+#### `allowLocalUrls` `COMPANION_ALLOW_LOCAL_URLS`
+
+A boolean flag to tell Companion whether to allow requesting local URLs (non-internet IPs).
+
+:::caution
+Only enable this in development. **Enabling it in production is a security risk.**
+:::
+
+#### `corsOrigins` `COMPANION_CLIENT_ORIGINS`
+
+Allowed CORS Origins (default `true`). 
+Passed as the `origin` option in [cors](https://github.com/expressjs/cors#configuration-options))
+
+#### `COMPANION_CLIENT_ORIGINS_REGEX`
+
+Like COMPANION_CLIENT_ORIGINS, but allows a single regex instead.
+`COMPANION_CLIENT_ORIGINS` will be ignored if this is used.
+
+#### `chunkSize` `COMPANION_CHUNK_SIZE`
+
+Controls how big the uploaded chunks are for AWS S3 Multipart and Tus.
+Smaller values lead to more overhead, but larger values lead to slower retries in case of bad network connections.
+Passed to tus-js-client [`chunkSize`](https://github.com/tus/tus-js-client/blob/master/docs/api.md#chunksize) as well as [AWS S3 Multipart](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html) `partSize`.
+
+### Standalone only options
+
+These environment variables apply only to Companion running in [standalone mode](#standalone).
+
```

Comment:
```suggestion

#### `REDIS_EXPRESS_SESSION_PREFIX`

Set a custom prefix for redis keys created by [connect-redis](https://github.com/tj/connect-redis). Defaults to `sess:`.  Sessions are used for storing authentication state and for allowing thumbnails to be loaded by the browser via Companion.

```

Reply:
Could we add a small why you would want to change this?